omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/PersonResource1_10.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_10;
+
+import io.swagger.models.Model;
+import org.openmrs.Person;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.annotation.Resource;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingCrudResource;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.PersonResource1_8;
+
+/**
+ * {@link Resource} for Person, supporting standard CRUD operations
+ */
+@Resource(name = RestConstants.VERSION_1 + "/person", order = 3, supportedClass = Person.class, supportedOpenmrsVersions = {
+        "1.10.4 - 1.10.6" })
```

Comment:
I'd think this should be 1.10.3

Reply:
@wluyima true, the confusion came from the fact that there is also a _branch_ 1.10.3 (??), and on that branch `birthtime` is not part of `Person` and @Andu033 was doing locally `git checkout 1.10.3`, this took him to the branch and not the release tag, hence the erroneous conclusion as to when `birthtime` got added.

@wluyima @dkayiwa can this branch be removed if it has not use?

@Andu033, sorry, would you mind submit another PR based on @wluyima's changes?

Reply:
@mks-d yes . But we must remove the branch first to make the birthtime exist in 1.10.3?

Reply:
The branch in itself has no impact, what matters is to figure out for sure whether `Person` in the released Core 1.10.3 (so the WAR coming out of the Maven repo) did contain `birthtime` or not.

Didn't you say that you even unpacked the WAR to check that out?

@dkayiwa / @wluyima is it possible that, somehow, Core 1.10.3 was released off this branch 1.10.3 and not off the _tag_ 1.10.3? There's something dodgy here.

Reply:
I have just taken a look at noticed that the 1.10.3 release was not done correctly. So a 1.10.3 branch and a 1.10.3 tag were created instead of just the 1.10.3 tag.

Reply:
>I have just taken a look at noticed that the 1.10.3 release was not done correctly.

@dkayiwa but do you know if it's the tag or the branch that represents what was actually packaged as a WAR on the Maven repo? I think @Andu033 did the test that when you (Maven) depend on 1.10.3 `Person.birthtime` isn't there to be found...

Reply:
Tags are what matter, in any case by convention true maintenance branches are named with a **.x** prefix e.g. 1.10.x, 1.11.x etc. therefore I think 1.10.3 must be a stray branch that was created by mistake.

Reply:
@mks-d even if no war file was created for the 1.10.3 tag which I strongly doubt, we can still create it at anytime since it exists, this is one of the primary reasons why tags are created in the first place, to be able to recreate archives at that particular revision in case need arises.

Reply:
To avoid the confusion, i have just deleted the erroneously created 1.10.3 branch. So we now have only the 1.10.3 tag which does not have the Person.birthtime field: https://github.com/openmrs/openmrs-core/blob/1.10.3/api/src/main/java/org/openmrs/Person.java

Reply:
Thanks @dkayiwa  . This means that no more changes are required , right?

Reply:
@wluyima, @Andu033 will confirm but I think that the released WAR 1.10.3 does not contain `Person.birthtime`. If that's indeed the case we know that the tag 1.10.3 is not the commit off which the release was made. So yes we could re-release a WAR out of that tag, but which version would that be?

Shouldn't we assume that the Maven reality is such that 1.10.3 out there do not contain `Person.birthtime` while 1.10.4 do.

---

@Andu033 yes something must still be done in regards to the 1.11.3 vs 1.11.4 situation.

Reply:
According to this ticket https://issues.openmrs.org/browse/TRUNK-4727 the birthtime field starts with 1.10.4

Am i missing something to expect it in 1.10.3?

Reply:
Looks like there was a mess around the 1.10.3 and yes it appears like the fix actually shows up in 1.10.4

Reply:
Really strange, from the git history the [backport](https://github.com/openmrs/openmrs-core/commit/cfc2f26a6e8042e8b9cb95b7fd53a44cf55f7f5c) happened between the 1.10.2 and 1.10.3 releases but for some reason the  fix doesn't show up in 1.10.3 tag. I'm suspicious about the 1.10.3 release.

Reply:
I now recall what happened, there was a security fix that was applied and we had special releases for the supported release lines which included 1.10.3, the releases were such that they only contained this fix and everything that had been backported to the respective maintenance branches got bumped to the next maintenance releases for each line. So yeah the commit was made earlier but was played forward due to these release hence this showing up in 1.10.4 and not 1.10.3.